lambdas/call-ticf-cri/src/main/java/uk/gov/di/ipv/core/callticfcri/service/TicfCriService.java

```diff
@@ -0,0 +1,133 @@
+package uk.gov.di.ipv.core.callticfcri.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jwt.SignedJWT;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.lambda.powertools.tracing.Tracing;
+import uk.gov.di.ipv.core.callticfcri.dto.TicfCriDto;
+import uk.gov.di.ipv.core.callticfcri.exception.TicfCriServiceException;
+import uk.gov.di.ipv.core.library.dto.BackEndCriConfig;
+import uk.gov.di.ipv.core.library.exceptions.VerifiableCredentialException;
+import uk.gov.di.ipv.core.library.persistence.item.ClientOAuthSessionItem;
+import uk.gov.di.ipv.core.library.persistence.item.IpvSessionItem;
+import uk.gov.di.ipv.core.library.service.ConfigService;
+import uk.gov.di.ipv.core.library.verifiablecredential.validator.VerifiableCredentialJwtValidator;
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static uk.gov.di.ipv.core.library.domain.CriConstants.TICF_CRI;
+
+public class TicfCriService {
```

Comment:
(Some day maybe this will become `BackendCriService`, but I think it's fine to keep it specific for now - we don't know how similar other such CRIs might be, or if we'll ever need them!)